torch/cuda/__init__.py

```diff
@@ -339,8 +339,8 @@ class StreamContext(object):
     def __init__(self, stream: Optional['torch.cuda.Stream']):  # type: ignore
         self.stream = stream
         # Initialize the below streams to default stream on the current device
-        self.src_prev_stream = torch.cuda.default_stream(None)
-        self.dst_prev_stream = torch.cuda.default_stream(None)
+        self.src_prev_stream = torch.cuda.default_stream(None) if stream else None
+        self.dst_prev_stream = torch.cuda.default_stream(None) if stream else None
```

Comment:
Can you try this? 

self.src_prev_stream = torch.cuda.default_stream(None)
self.dst_prev_stream = torch.cuda.default_stream(None)
        if not torch.jit.is_scripting():
            self.src_prev_stream = None
            self.dst_prev_stream = None
See if this works for the buck test?

Reply:
Tried this fix and it works locally. I couldn't run the buck test.